assets/app/scripts/services/projects.js

```diff
@@ -13,17 +13,23 @@ angular.module('openshiftConsole')
           return  AuthService
                     .withUser()
                     .then(function() {
+                      // basic compatibility w/previous impl with a controller
                       var context = {
                         // TODO: swap $.Deferred() for $q.defer()
+                        projectPromise: $.Deferred(),
                         projectName: projectName,
-                        projectPromise: $.Deferred()
+                        project: undefined,
+                        projects: {}
```

Comment:
is this context being reused somewhere that would populate projects?  i thought the project nav made its own request to get projects now outside of the main scope.